modules/utils.py

```diff
@@ -0,0 +1,85 @@
+# turns a letter into the morse code equivalent
+# or `None` for an invalid input
+def letterToMorse(letter):
+    LETTER_TO_MORSE = {
+        "a": ".-",
+        "b": "-...",
+        "c": "-.-.",
+        "d": "-..",
+        "e": ".",
+        "f": "..-.",
+        "g": "--.",
+        "h": "....",
+        "i": "..",
+        "j": ".---",
+        "k": "-.-",
+        "l": ".-..",
+        "m": "--",
+        "n": "-.",
+        "o": "---",
+        "p": ".--.",
+        "q": "--.-",
+        "r": ".-.",
+        "s": "...",
+        "t": "-",
+        "u": "..-",
+        "v": "...-",
+        "w": ".--",
+        "x": "-..-",
+        "y": "-.--",
+        "z": "--..",
+    }
+    if letter in LETTER_TO_MORSE.keys():
+        return LETTER_TO_MORSE[letter]
+    return None
+
+# converts a whole string into morse code
+def stringToMorse(string):
+    words = string.split(" ")
+    letters = map(list,words)
+    lettersConverted = map(lambda x:map(letterToMorse,x),letters)
+    wordsConverted = map(lambda x:" ".join(x),lettersConverted)
+    return " // ".join(wordsConverted)
+
+# turns a morse code string  equivalent
+# or `None` for an invalid input
+def morseToLetter(morse):
+    MORSE_TO_LETTER = {
+        ".-": "a",
+        "-...": "b",
+        "-.-.": "c",
+        "-..": "d",
+        ".": "e",
+        "..-.": "f",
+        "--.": "g",
+        "....": "h",
+        "..": "i",
+        ".---": "j",
+        "-.-": "k",
+        ".-..": "l",
+        "--": "m",
+        "-.": "n",
+        "---": "o",
+        ".--.": "p",
+        "--.-": "q",
+        ".-.": "r",
+        "...": "s",
+        "-": "t",
+        "..-": "u",
+        "...-": "v",
+        ".--": "w",
+        "-..-": "x",
+        "-.--": "y",
+        "--..": "z",
+    }
```

Comment:
This could be derived in a single line from the previous dictionary. (make sure to do it outside of the function so that it's not redone every time the function is called)